### DISCUSSION TITLE
Ability to turn off Context::class metadata in log

### DISCUSSION THREAD

**Discussion by lem62** [laravel/framework#53990] [Ideas]
Topic - _Illuminate\Support\Facades\Context_

Context feature append metadata in every log record. It will be useful to get ability to hidden Context metadata in log (in some case).

Reason:
- Large log files
- Redundant information in analysis

**Comment by henzeb** [comment]
Let me get this straight. You want to use the context data,  but only when you see fit?

Like log()->withContext();

Or log()->withoutContext();

So that context is only shown in log entries you want?

Because hidden context is a part used to share context between components without showing up in the logs at all...
